docs/Producing.md

```diff
@@ -110,6 +112,7 @@ For instance:
 
 PRODUCER keyed-message
 topic: fakerjs-topic
+headers: messageId={{random.uuid}}

```

Comment:
could you add a new hearder parameter to see that comma isused please.

Reply:
Added second header